flow/util/utils.mk

```diff
@@ -15,7 +15,7 @@ update_metadata: $(REPORTS_DIR)/metadata-$(FLOW_VARIANT).json
 	      $(DESIGN_DIR)/metadata-$(FLOW_VARIANT)-ok.json
 
 update_rules:
-	$(UTILS_DIR)/genRuleFile.py $(DESIGN_DIR) $(FLOW_VARIANT)
+	$(UTILS_DIR)/genRuleFile.py $(DESIGN_DIR) --variant $(FLOW_VARIANT) --update
```

Comment:
Change to -failing -tighten as discussed.

Reply:
Fixed on [17395ab](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/453/commits/17395ab20bf565656c2f980b0cf5148e3346fa87)